App.tsx

```diff
@@ -18,38 +18,30 @@ import {
   StatusBar,
 } from 'react-native';
 
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
-
-import {Hello} from './components/Hello';
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+import {Button} from './components/Button';
+import {FormTextInput} from './components/FormTextInput';
 
 const App = () => {
-  const usingHermes = typeof HermesInternal === 'object' && HermesInternal !== null;
-
   return (
     <Fragment>
       <StatusBar barStyle="dark-content" />
       <SafeAreaView>
         <ScrollView
           contentInsetAdjustmentBehavior="automatic"
           style={styles.scrollView}>
-          <Header />
-          { usingHermes &&
-            <View style={styles.engine}>
-              <Text style={styles.footer}>Engine: Hermes</Text>
-            </View>
-          }
-          <View style={styles.sectionContainer}>
-            <Text style={styles.sectionTitle}>My Awesome Project</Text>
+          <View style={styles.sectionHeader}>
+            <Text style={styles.sectionTitle}>Bem vindo(a) à Taqtile!</Text>
           </View>
           <View style={styles.body}>
-            <View style={styles.sectionContainer}>
-              <Hello name="World" enthusiasmLevel={1} />
+            <View style={styles.container}>
+              <FormTextInput label="E-mail" />
+            </View>
+            <View style={styles.container}>
+              <FormTextInput label="Senha" />
```

Comment:
Já que você já componentizou o form, deixa eu perguntar: será que em todos os lugares que você for usar um form essa `<View style={styles.container}>` vai vir em volta?

Se a resposta dessa pergunta for sim, isso é um bom indício de que essa view poderia estar dentro do componente. O que acha?

Reply:
Faz sentido. Passei esse "wrapper" para dentro do FormItem (ex-FormTextInput). Fiz a mesma coisa para o Button.